configuration/default-applications.md

```diff
@@ -0,0 +1,107 @@
+
+	$ xdg-mime default zathura-pdf-poppler.desktop application/pdf
+
+`xdg-mime` confirms:
+
+	$ xdg-mime query default application/pdf
+	zathura-pdf-poppler.desktop
+
+This setting is usually effective immediately and is written to `~/.local/share/mimeapps.list`, i.e.:
+
+	[Default Applications]
+	application/pdf=zathura-pdf-poppler.pdf
+
+See below for a more extensive `mimeapps.list` file.
+
+(If you want to apply this setting for every AppVM of a TemplateVM you need to copy `mimeapps.list` to `/usr/share/applications/` in the TemplateVM)
+
+
+## Setting `qvm-open-in-(d)vm` as a default application for file types/URLs
+
+Desktop entry files contain, in essence, instructions for executing a command. This turns out to be quite useful in combination with Qubes' [`qvm-open-in-dvm`](https://www.qubes-os.org/doc/vm-tools/qvm-open-in-vm/) and [`qvm-open-in-dvm`](https://www.qubes-os.org/doc/vm-tools/qvm-open-in-dvm/), which open a specified file (or URL) in a new DispVM or in a specific VM, respectively.
+
+By creating a Desktop entry file and setting it as the standard application for one or more MIME types you can configure a VM to open these kind of files in either a DispVM or a specified VM. (Substitute `qvm-open-in-dvm` with  `qvm-open-in-vm vmname` [e.g. `qvm-open-in-vm work`] in any command, file or example below if that is what you want to achieve.)
+
+In the VM(s) you want to run `qvm-open-in-dvm` from (not in the VM you want the file/URL to be opened): Add or edit entries in `~/.local/share/applications/mimeapps.list` so that `qvm-open-in-dvm.desktop` is the default application for the MIME types you specify. You cannot use wildcards in MIME types. Example:
+
+	
+	[Default Applications]
+	x-scheme-handler/unknown=qvm-open-in-dvm.desktop
+	x-scheme-handler/about=qvm-open-in-dvm.desktop
+	x-scheme-handler/http=qvm-open-in-dvm.desktop
+	x-scheme-handler/https=qvm-open-in-dvm.desktop
+	text/html=qvm-open-in-dvm.desktop
+	text/xml=qvm-open-in-dvm.desktop
+	image/gif=qvm-open-in-dvm.desktop
+	image/jpeg=qvm-open-in-dvm.desktop
+	image/png=qvm-open-in-dvm.desktop
+	application/xhtml+xml=qvm-open-in-dvm.desktop
+	application/xml=qvm-open-in-dvm.desktop
+	application/vnd.mozilla.xul+xml=qvm-open-in-dvm.desktop
+	application/rss+xml=qvm-open-in-dvm.desktop
+	application/rdf+xml=qvm-open-in-dvm.desktop
+	application/msword=qvm-open-in-dvm.desktop
+	application/vnd.oasis.opendocument.spreadsheet=qvm-open-in-dvm.desktop
+	application/vnd.oasis.opendocument.text=qvm-open-in-dvm.desktop
+	application/pdf=qvm-open-in-dvm.desktop
+
+Still in the VM(s) you want to run `qvm-open-in-dvm` from: You need to create a corresponding `.desktop` file and save it in `/usr/share/applications/` or `~/.local/share/applications/`, depending on what you prefer. Every MIME type assigned to this application in `mimeapps.list` must be listed in the `.desktop` file, but fortunately wildcards can be used for subtypes. Example:
```

Comment:
Here should be a note about difference between `/usr/share/applications/` and `~/.local/share/applications/`. The former should be in TemplateVM, the latter in AppVM. Currently the text suggests it doesn't matter.